suite-native/device/src/hooks/useDeviceConnect.tsx

```diff
@@ -79,7 +82,7 @@ export const useDeviceConnect = () => {
                 ),
             });
         }
-    }, [isConnectedDeviceUninitialized, showAlert, translate]);
+    }, [isConnectedDeviceUninitialized, showAlert, translate, selectedDevice]);
```

Comment:
this seems to be unused in effect

Reply:
https://github.com/trezor/trezor-suite/pull/9694/commits/c9f9b0587b4ba3bea0645ae010e508e0e0911fc4